example_publisher/publisher.py

```diff
@@ -141,6 +142,7 @@ async def on_notify_price_sched(self, subscription: int) -> None:
         await self.pythd.update_price(
             product.price_account, scaled_price, scaled_conf, TRADING
         )
+        self.last_successful_update = price.timestamp
```

Comment:
I think it is safer to get a maximum here.

Reply:
Make sense. Fixed.